### PR TITLE
AEA-2143 Refactor EPSAT test-pack upload code from one file into logical components

### PR DESCRIPTION
**Pull request by booshibooshi**

### CODE REVIEW

File: tool/site/client/src/services/test-packs/communicationRequests.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      recipient: [
+        {
+          type: "Patient",
+          identifier: {
+            system: "https://fhir.nhs.uk/Id/nhs-number",
+            value: getNhsNumber(patientEntry)
+          }
+        }
+      ]
+    } as fhir.CommunicationRequest
```
Comment (JackRobinsonIW) [inline tool/site/client/src/services/test-packs/communicationRequests.ts:33]: just wondering why we need this casting here? are our underlying types not quite right?

File: tool/site/client/src/services/test-packs/index.ts [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+      return "prescribing-cost-centre-0101"
+    case "0104":
+    case "0105":
+    case "0108":
+    case "0125":
+      return "prescribing-cost-centre-non-0101"
+    default:
+      throw new Error("Prescription type not handled")
+  }
+}
```
Comment (JackRobinsonIW) [inline tool/site/client/src/services/test-packs/index.ts:257]: I've been through this with as much attention as I can manage, looks good but it is possible I've missed something

File: tool/site/client/src/services/test-packs/communicationRequests.ts (hunk 1 above)
Comment (booshibooshi) [reply tool/site/client/src/services/test-packs/communicationRequests.ts:33]: It's the fhir types here, bundle.entry is a derived fhirResource type so could be any number of a lot of resource types, using this to add additional type safety to our resource building

File: tool/site/client/src/services/test-packs/practitionerRoles.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import * as fhir from "fhir/r4"
+// import {PrescriberType} from "."
```
[the remark this answers is not in the document]
Comment (booshibooshi) [reply tool/site/client/src/services/test-packs/practitionerRoles.ts:2]: this is a placeholder, will be including in future PR

**Review by JackRobinsonIW — APPROVED** [review]

**Pull request merged**